lib/configurator.js

```diff
@@ -7,6 +7,8 @@ const extractTextPluginCompilerName = 'extract-text-webpack-plugin';
 const isomorphicSpriteModule = 'svg-sprite-loader/runtime/sprite.build';
 const isomorphicSymbolModule = 'svg-baker-runtime/symbol';
 
+const isTargetBrowser = target => target === 'web' || target === 'electron-renderer';
```

Comment:
There is also `electron-main` target. I think easiest way is to test `/node/` regexp against the target string.